src/content/docs/journey-migration/making-the-switch.mdx

```diff
@@ -0,0 +1,127 @@
+---
+title: "Making the switch"
```

Comment:
I think this title needs to be more specific, alone it doesn't tell me what I'm switching from or to or when to use thiw particular doc. If you assume this doc will be the only link they see in a list of google results, what helps them click it?

Reply:
Fixed!